amethyst_rendy/src/lib.rs

```diff
@@ -48,6 +48,12 @@
 #[macro_use]
 extern crate amethyst_derive;
 
+#[macro_use]
+extern crate derive_new;
```

Comment:
This should not be required. We use rust 2018, just import the macro where you use it.

Reply:
I thought line "make the macros globally accessible in the crates" was about this, is it not?

Reply:
The reasoning behind this was to make it globally accessible in the crate. I think this was mentioned in PR #1843 where it was suggested to switch from importing the macro in each file to this method. I can change it back though if that's what the amethyst team prefers.

Reply:
Well, I guess this is not the right place to complain about it then. I generally prefer to not have `extern crate`, but it already happened and we might need a separate pass to discuss and deal with this globally. So I'd say keep it for now.